### PR TITLE
Mailet 129

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: server/mailet/mailets/src/test/java/org/apache/james/transport/matchers/RemoteAddrInNetworkTest.java [hunk 1]
```diff
@@ -18,51 +18,60 @@
...
+                .build();
     }
 
-    // test if no recipient get returned cause it not match
     @Test
-    public void testRemoteAddrInNetworkNotMatch() throws MessagingException {
-        setRemoteAddr("192.168.1.1");
-
-        setupAll();
+    public void matchShouldMatchWhenOnSameNetwork() throws MessagingException {
```
Comment (mbaechler) [inline server/mailet/mailets/src/test/java/org/apache/james/transport/matchers/RemoteAddrInNetworkTest.java]: you can omit match IMO

Comment (rouazana) [inline server/mailet/mailets/src/test/java/org/apache/james/transport/matchers/RemoteAddrInNetworkTest.java]: done

**Comment by aduprat** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Pull request merged**